flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/functions/sql/SqlFirstLastValueAggFunction.java

```diff
@@ -45,7 +44,7 @@ public SqlFirstLastValueAggFunction(SqlKind sqlKind) {
 				sqlKind,
 				ReturnTypes.ARG0_NULLABLE_IF_EMPTY,
 				null,
-				OperandTypes.or(OperandTypes.ANY, OperandTypes.family(SqlTypeFamily.ANY, SqlTypeFamily.ANY)),
+				OperandTypes.ANY,
```

Comment:
Could you align the implementation with `org.apache.calcite.sql.fun.SqlFirstLastValueAggFunction`? The only different should be `requirsOver`, and would be great to mention it in the class javadoc.